smile_id_core/Utilities.py

```diff
@@ -207,24 +206,18 @@ def execute_post(url, payload):
 
 
 def validate_sec_params(sec_key_dict: Dict):
-    if not sec_key_dict.get("sec_key") and not sec_key_dict.get("signature"):
-        raise Exception("Missing key, must provide a 'sec_key' or 'signature' field")
+    if not sec_key_dict.get("signature"):
+        raise Exception("Missing key, must provide a 'signature' field")
     if not sec_key_dict.get("timestamp"):
         raise Exception("Missing 'timestamp' field")
 
 
-def get_signature(partner_id, api_key, is_signature):
+def get_signature(partner_id, api_key):
     sec_key_gen = Signature(partner_id, api_key)
     sec_key_object = (
         sec_key_gen.generate_signature()
-        if is_signature
-        else sec_key_gen.generate_sec_key()
     )
-    sec_key = sec_key_object.get("sec_key")
     signature = sec_key_object.get("signature")
     payload = {"timestamp": sec_key_object["timestamp"]}
-    if sec_key:
-        payload.update({"sec_key": sec_key})
-    else:
-        payload.update({"signature": signature})
+    payload.update({"signature": signature})
```

Comment:
since you're removing the if condition you can as well add the signature key in the dict in line 221
```py
    payload = {"timestamp": sec_key_object["timestamp"],"signature": signature }
```